src/components/Selector/Selector.js

```diff
@@ -17,23 +17,35 @@ import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
 import { css } from '@emotion/core';
 import withAriaSelected from '../../util/withAriaSelected';
+import { shadowSingle } from '../../styles/style-helpers';
 
 const baseStyles = ({ theme }) => css`
   label: selector;
   cursor: pointer;
-  padding: ${theme.spacings.mega};
-  border-radius: ${theme.borderRadius.mega};
-  border: 1px solid ${theme.colors.n300};
-  background-color: 1px solid ${theme.colors.n100};
+  padding: ${theme.spacings.giga};
+  border-radius: ${theme.borderRadius.giga};
+  border: ${theme.borderWidth.kilo} solid ${theme.colors.n300};
+  background-color: ${theme.colors.white};
   margin-bottom: ${theme.spacings.mega};
+  fill: ${theme.colors.n400};
 `;
 
+const hoverStyles = ({ selected, theme }) =>
+  !selected &&
+  css`
+    label: selector--hover;
+    & :hover {
```

Comment:
That looks like a typo 🧐
```suggestion
    &:hover {
```